doc/content/gateways/gateway-claiming/claim-gateways.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Claim a Gateway"
+description: ""
+weight: 2
+distributions: ["Enterprise", "Cloud"]
+--- 
+
+You can update the example below with your tenant ID and cluster ID by filling them here.
+
+{{< tenant-cluster-selector >}}
+
+## Prerequisites
+
+1. An account in The Things Industries Cloud.
+2. EUI of the gateway that you wish to claim.
+3. The Claim Authentication Code for the gateway (shared out of band by the current gateway owner).
+4. The Current CUPS Key used by the gateway (for {{% lbs %}} only).
+
+## General
```

Comment:
```suggestion
## Note on Using the Gateway ID
```

Reply:
This is an `H2` section that contains 2 H3 sections for LBS and non-LBS so I don't think `Note on Using the Gateway ID` is the correct heading.

Reply:
Ok I get it. Although it makes more sense to change other sections to H2, because they contain the most important info, and put this as a note. Doesn't really make sense to start with explaining the flags that are optional